### PR TITLE
Ncnews 7

### PR DESCRIPTION
**Pull request by Ha-Mundo**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -163,3 +163,50 @@ describe("GET /api/users", () => {
...
+  });
+  test("200: should respond with correct number of comments for the given article id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(res => {
+        expect(res.body.article.comment_count).toBe("11");
+      });
+  });
+  test("200: should respond with correct number of comments for the given article id", () => {
```
Comment (dappernerddesigns) [inline __tests__/app.test.js]: What do we gain by repeating this test three times?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -163,3 +163,50 @@ describe("GET /api/users", () => {
...
+    return request(app)
+      .get("/api/articles/9")
+      .expect(200)
+      .then(res => {
+        expect(res.body.article.comment_count).toBe("2");
+      });
+  });
+});
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:204]: There is no test for an article with no comments. This should be returning an empty array with no errors

**Review by dappernerddesigns — CHANGES_REQUESTED** [review]
Please read the notes on your review and add the appropriate test, then remove the duplicates. Re submit when ready.

File: __tests__/app.test.js (hunk 2 above)
Comment (Ha-Mundo) [reply __tests__/app.test.js:212]: Ok.
 I will remove that useless test
 I'm sorry about that

File: __tests__/app.test.js (hunk 1 above)
Comment (Ha-Mundo) [reply __tests__/app.test.js]: OK Right.
I will remove the unnecessary tests
